consensus/consensus.go

```diff
@@ -167,8 +162,6 @@ func NewConsensus(
 			if err != nil {
 				log.Logger("consensus").Info("Failed to commit the block.", zap.Error(err), zap.Uint64("height", blk.Height()))
 			}
-			// Remove transfers in this block from ActPool and reset ActPool state
-			ap.Reset()
```

Comment:
this is removed, does not to be called somewhere else?

Reply:
i mean, i only see it being called in _test.go

Reply:
I have same question as Dustin. Reset checks current account nonce and clears out processed actions.
i think maybe we can combine reset pool with delete invalid action

Reply:
checking